front/src/modules/trainschedule/components/ImportTrainSchedule/ImportTrainScheduleConfig.tsx

```diff
@@ -203,13 +199,40 @@ const ImportTrainScheduleConfig = ({
         const formattedArrivalTime = `${startDate} ${arrivalTime}`;
         const formattedDepartureTime = `${startDate} ${departureTime}`;
 
+        let stopFor: number | undefined;
+
+        const arrivalDate = new Date(`${startDate}T${arrivalTime}`);
+        const departureDate = new Date(`${startDate}T${departureTime}`);
+        if (ocpType === 'stop') {
+          if (arrivalTime && departureTime) {
+            if (arrivalTime === departureTime) {
+              stopFor = 0;
+            } else {
+              stopFor = Math.round((departureDate.getTime() - arrivalDate.getTime()) / 1000);
+            }
```

Comment:
If arrivalTime and departureTime are the same, then stopFor will be 0 ;) You can remove the first test
```suggestion
            stopFor = Math.round((departureDate.getTime() - arrivalDate.getTime()) / 1000);
```